### DISCUSSION TITLE
Question about training loss in train_dreambooth_sd3.py

### DISCUSSION THREAD

**Discussion by ningpingwang** [huggingface/diffusers#10618] [Q&A]
Here is the code snippet of computing training loss `diffusers/examples/dreambooth/train_dreambooth_sd3.py`
```
noisy_model_input = (1.0 - sigmas) * model_input + sigmas * noise
model_pred = transformer(noisy_model_input, ...)
model_pred = model_pred * (-sigmas) + noisy_model_input
target = model_input
loss = torch.mean(
    (weighting.float() * (model_pred.float() - target.float()) ** 2).reshape(target.shape[0], -1),
    1,
)
```

In the final step, `model_pred` is `sigmas * (noise - vanilla_model_pred) + (1.0 - sigmas) * model_input`, 
so `model_pred - target` is  `sigmas * (noise - vanilla_model_pred)  - sigmas * model_input`.
But from my understanding of flow match loss, it should be `sigmas * (noise - vanilla_model_pred)` without the term ` - sigmas * model_input`.

Am I missing something?
